museum/importLocationHistory

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
```

Comment:
again file is copypasta. Next time I do that I'll abstract it out into a function or 6. I promise.